### ISSUE TITLE
"sp_" in function names is not necessary

### ISSUE THREAD

**Issue by hrlai** [xp-song/allometree#3]
@xp-song should we just simplify the function names to go without the `sp_` prefix?

I'm also trying to think of a friendly name for the mixed-effect model function. This can be too long, but what about:
- `allometree_single` for single `lm`s
- `allometree_single_multi` for multiple `lm`s at once
- `allometree_mixed_effects` or `allometree_mlm` for mixed-effects or multilevel models (MLM) ?

**Comment by xp-song** [comment]
Thanks for mentioning this. So initially I used "sp_" to denote single-species models, but to make it clear I changed all "sp_" to "ss_". Then, for the mixed-effects models, I'll change them to start with "mix_". 

I think that consistent prefix helps differentiate between the 2 immediately, rather than having the package name as the prefix!

If this is okay, I will make the changes and push them in the next commit
